flepimop/gempyor_pkg/src/gempyor/parameters.py

```diff
@@ -224,7 +230,9 @@ class via `subpop_names`.
 
         for idx, pn in enumerate(self.pnames):
             if "dist" in self.pdata[pn]:
-                param_arr[idx] = np.full((n_days, nsubpops), self.pdata[pn]["dist"]())
+                param_arr[idx] = np.full(
+                    (n_days, nsubpops), self.pdata[pn]["dist"].sample().item()
```

Comment:
with the change to callability, can this now be

```suggestion
                    (n_days, nsubpops), self.pdata[pn]["dist"]()
```